### ISSUE TITLE
Opening two documents at the same time on the same tab breaks the first one.

### ISSUE THREAD

**Issue by tsmgodoi** [ONLYOFFICE/DocumentServer#1382]
**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
If I open one document on one tab and then open another document on the same tab but on another editor (another element) it breaks the first one, like in the picture below:
![image](https://user-images.githubusercontent.com/14880633/129639645-8cbcdeaa-7954-4108-ad98-bbd72461673a.png)
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Open two editors at the same time on the same tab. The first document will have an overlay over the text, like in the picture above.
**What is the expected behavior?**
The first document should be displayed without an overlay over the text.
**Did this work in previous versions of DocumentServer?**
No.
**DocumentServer version:**
5.6.5 or 6.3.1
**Operating System:**
Windows 10 20H2 (OnlyOffice Document Server running under Docker on a Linux environment or on any docker environment)
**Browser version:**
Google Chrome 91.0.4472.114

**Comment by ShockwaveNN** [comment]
This may be problem with chrome hardware acceleration

Could you try to disable it?

Could you try to reproduce this problem on another PC? usually this problem exists only on some specific combination of OS, Drivers and Video Card

**Comment by tsmgodoi** [comment]
I disabled Google Chrome hardware acceleration and tried on another PC. The same thing happens. The same thing happens on Mozilla Firefox.
Is opening two editors at the same time supported? I see there's the following comment on the file api.js:
`    // TODO: allow several instances on one page simultaneously

    DocsAPI.DocEditor = function(placeholderId, config) {
        var _self = this,`

**Comment by ShockwaveNN** [comment]
Oh, I read this incorrectly

You've trying to open two same document in one tab, not in two different

I think there may be a bug and you're right
Could you provide some test example of this page?

Also as a side note - why your icons on TopToolbar are different?
Default are and I think it's uncofigurable

![image](https://user-images.githubusercontent.com/668524/129791149-6b56adde-c4be-401f-bca8-e163c72fea3d.png)

Is this some kind of bug or your low level customization?

**Comment by tsmgodoi** [comment]
Yes, we're trying to load two documents on the same tab. We have an editor where the user is writing the text, but while he's writing the text he can search other texts and view it also using OnlyOffice. When he go back to the original text there's an overlay over it.
You can watch it happening here: https://youtu.be/GWzx5T77jbs?t=66

About the top toolbar...
It's not hard to customize it using plugins. The anonymous function passed to callCommand is executed inside the editor, so we just have to hide and/or add elements to the DOM. We know that kind of customization is prone to break on updates, so we're at our own risk here. So we won't complain if something breaks there. We use it only inside our organization and our users really need to customize it instead of putting those plugins inside the Plugins tab. (We have a corporate license, by the way)

**Comment by ShockwaveNN** [comment]
Two question about two editors

1. Are they isolated each in each own iframe
2. Could you provide example of such integration, maybe simplest html page will do good? I'm almost sure that you're right and there is some kind bug in this, but this save me a lot of time if we pin this issue to simple html page


About customization - got it, don't remember that anyone do that before, but I think it's a good way to fully customize specific version

**Comment by tsmgodoi** [comment]
Well, while I was developing a demo of the problem I concluded that it does work if you use different placeholder's ids. That behavior only happens if you use the same placeholder id for the two editors. With that I realized that in our system we were using the same placeholder id. I changed it to generate a random id for the placeholder for each editor and it worked as expected.
I'll be closing this issue. Thank you for the support!
